.github/workflows/foreman_plugin.yml

```diff
@@ -29,9 +29,13 @@ jobs:
     steps:
       - name: Build test matrix
         id: build_matrix
-        # TODO: Make this dynamic, based on the Foreman version
         run: |
-          echo 'matrix={"ruby": ["2.7"], "node": ["12"]}' >> "$GITHUB_OUTPUT"
+          {
+            echo 'matrix<<EOF'
```

Comment:
```suggestion
            echo 'matrix=<<EOF'
```

Reply:
I also do find the syntax odd, and yet it is correct:
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings

(tested here: https://github.com/evgeni/gha-playground/actions/runs/6544062561)

Reply:
Ah yes, I remember going over this part of the docs but I've never used it before.